internal/renderer/inspect.go

```diff
@@ -14,7 +15,7 @@ import (
 )
 
 // Inspect dumps the metadata of an app
-func Inspect(appname string) error {
+func Inspect(out io.Writer, appname string) error {
```

Comment:
I think you wanted to use it with `w := tabwriter.NewWriter(os.Stdout`

Reply:
oh good point, forgot to do something in there :sweat_smile: